pkg/rancher-desktop/utils/version.ts

```diff
@@ -25,6 +25,10 @@ async function getDevVersion() {
 }
 
 export async function getVersion() {
+  if (process.env.MOCK_VERSION) {
```

Comment:
> As a general rule, we prefix our environment variables with `RD_`. I think we should strive to be consistent with this naming convention.
> 
> We might want to follow up in another PR to rename `MOCK_FOR_SCREENSHOTS` so that it fits this convention as well.

Thanks Phil, updated